src/shared/cpu-set-util.c

```diff
@@ -290,3 +337,22 @@ int cpu_set_from_dbus(const uint8_t *bits, size_t size, CPUSet *set) {
         *set = TAKE_STRUCT(s);
         return 0;
 }
+
+int cpu_mask_add_all(CPUSet *mask) {
+        long m;
+
+        assert(mask);
+
+        m = sysconf(_SC_NPROCESSORS_ONLN);
+        assert_se(m > 0);
+
+        for (int i = 0; i < m; i++) {
+                int r;
+
+                r = cpu_set_add(mask, i);
```

Comment:
cpu_set_add() wants an `unsigned` as type for the 2nd arg. sysconf() returns a `long`. But you count in `int`. That's just confusing. Please try to stick to the minimal combinations of types. Or in other words, change `int i` to `unsigned i` please